### ISSUE TITLE
Tabbed / Stacked layouts

### ISSUE THREAD

**Issue by theduke** [Drakulix/fireplace#20]
Very cool project!

Do you plan to add support for tabbed / stacked layouts supported by `i3` and (recently) `sway`?

**Comment by Drakulix** [comment]
Yes I do. But I have some more immediate issues to take care of at first and this requires careful planning. The files responsible for i3-layouts are already growing quite large and I need to think about a way to refactor that code, if I add this feature. Otherwise this will be become a huge pain to maintain and most likely a buggy experience.
